api/main.py

```
@@ -0,0 +1,382 @@
+        )
+    except Exception as e:
+        logger.error(f"Error getting metrics: {e}")
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+@app.get("/api/visualization", response_model=VisualizationDataResponse)
+async def get_visualization_data():
+    """Get 3D visualization data"""
+    try:
+        g = get_graph()
+        positions, asset_ids, asset_colors, asset_text, edges_xyz = g.get_3d_visualization_data()
```

Ruff (0.14.2)

Unpacked variable `asset_text` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)